test/check-machines-settings

```diff
@@ -643,13 +648,13 @@ class TestMachinesSettings(machineslib.VirtualMachinesCase):
         b.wait_in_text("#vm-subVmTest1-system-state", "Running")
 
         # Test configuring watchdog for running VM
-        setWatchdogActionLive(action="reset")
+        setWatchdogActionLive(action="reset", removable=False)
```

Comment:
Is this because it's q35, or because it's running, and watchdogs can't be removed from a running VM?

Reply:
This is because there isn't any watchdog yet. The dialog just shows "Add" and no "Remove", obvsly.

Reply:
Thanks -- please add this as a comment.

Reply:
I have added another flag, "add", that will do the right thing. I think that is even better.